omnigibson/prims/xform_prim.py

```diff
@@ -145,6 +133,7 @@ def _set_xform_properties(self):
             xform_op_rot = lazy.pxr.UsdGeom.XformOp(self._prim.GetAttribute("xformOp:orient"))
         xformable.SetXformOpOrder([xform_op_translate, xform_op_rot, xform_op_scale])
 
+        # TODO: This is the line that causes Transformation Change on... errors. Fix it.
         self.set_position_orientation(position=current_position, orientation=current_orientation)
         new_position, new_orientation = self.get_position_orientation()
         r1 = R.from_quat(current_orientation).as_matrix()
```

Comment:
should we use `T.quat2mat()`?

Reply:
Do you mean this line: `r1 = R.from_quat(current_orientation).as_matrix()`